cantabular-import/dp-cantabular-xlsx-exporter.yml

```diff
@@ -0,0 +1,31 @@
+version: '3.3'
+services:
+    dp-cantabular-xlsx-exporter:
+        build:
+            context: ../../dp-cantabular-xlsx-exporter
+            dockerfile: Dockerfile.local
+        command:
+            - reflex
+            - -d
+            - none
+            - -c
+            - ./reflex
+        volumes:
+            - ../../dp-cantabular-xlsx-exporter:/dp-cantabular-xlsx-exporter
+        depends_on:
+            - kafka
+            - minio
```

Comment:
Also, please add these dependencies:
            - dp-dataset-api
            - dp-cantabular-csv-exporter

Reply:
I don't think that's appropriate.. the depends_on clause isn't about which order the services are called when in use but which order they need to start up in.. Notice none of our other services depend on each other in that way, this will just cause the start-up to take longer unnecessarily